plugins/modules/x509_crl.py

```diff
@@ -51,8 +51,17 @@
               I(revoked_certificates), but can also contain other revoked certificates. If the CRL file
               already exists, all entries from the existing CRL will also be included in the new CRL.
               When using C(update), you might be interested in setting I(ignore_timestamps) to C(true).
+            - The default value is C(generate).
```

Comment:
And another possible update here:
```suggestion
            - The default value is C(generate).
            - This parameter was called I(mode) before community.crypto 2.13.0. It has been renamed to avoid
              a collision with the common I(mode) parameter for setting the CRL file's access mode.
```